pytripgui/model/main_model.py

```diff
@@ -23,7 +22,7 @@ def __init__(self, app):
         self.vdx = None  # Only one VDX is allowed
         self.plans = []
         self.dos = []   # TODO: change to self.doss (plural s)
-        self.let = []   # TODO: change to self.lets (plural s)
+        self.lets = LetContainer()
```

Comment:
could also be 
let_list = 
when plural s doesn't work well. (in particular for dos, where plural s would be ambigous.)

Reply:
right :D